user/tests/wiring/serial_loopback2/loopback.cpp

```diff
@@ -19,15 +19,45 @@
 #include "unit-test/unit-test.h"
 #include "random.h"
 
-void runLoopback(size_t buffer_size_min, size_t buffer_size_max, bool sleep) {
+#ifndef USE_BUFFER_SIZE
+#warning "Using default 64 byte buffer"
+#define USE_BUFFER_SIZE (SERIAL_BUFFER_SIZE)
+#else
+hal_usart_buffer_config_t __attribute__((weak)) acquireSerial1Buffer()
```

Comment:
`__attribute__((weak))` needs to be removed?

Reply:
:+1: fixed